examples/visual_studio/buck2_utils/install.ps1

```diff
@@ -29,7 +29,8 @@ begin
 			[parameter(Mandatory=$true)] [String] $SourcePath,
 			[parameter(Mandatory=$true)] [String] $TargetPath
 		)
-		Copy-Item $SourcePath -Destination $TargetPath
+		New-Item -ItemType File -Path $TargetPath -Force | Out-Null
+		Copy-Item $SourcePath -Destination $TargetPath | Out-Null
```

Comment:
Because Copy-Item doesn't create the target directory if it doesn't exist: https://stackoverflow.com/questions/7523497/should-copy-item-create-the-destination-directory-structure
For example the first time the install script is ran, the target "$InstallDirectory" does not exist which makes Copy-Item fail when it tries to copy "main.exe" to "install/main.exe".